src/main/javascript/src/views/assignment/TerracottaBuilder.vue

```diff
@@ -222,6 +232,7 @@ export default {
       updateQuestion: "assessment/updateQuestion",
       deleteQuestion: "assessment/deleteQuestion",
       updateAnswer: "assessment/updateAnswer",
+      orderQuestions: "assessment/orderQuestions"
```

Comment:
I'm not seeing orderQuestions implementation, or where it is used?

Reply:
That was a mistake, I didn't mean to leave it in. I was debating lifting the logic up into a vuex action but ended up going another route.

Fixed